packages/web/components/elements/LabelsPicker.tsx

```diff
@@ -8,6 +8,11 @@ import { EditLabelChip } from './EditLabelChip'
 import { LabelsDispatcher } from '../../lib/hooks/useSetPageLabels'
 import { EditLabelChipStack } from './EditLabelChipStack'
 
+// AutosizeInput is a Class component, but the types are broken in React 18.
```

Comment:
Yeah in the extension we basically just re-created this functionality fairly easily, so we could probably do the same here.